docs/en/enterprise-edition/policy-reference/azure-policies/azure-logging-policies/bc-azure-2-53.adoc

```diff
@@ -0,0 +1,50 @@
+
+== Azure Synapse Workspace does not have extended audit logs
```

Comment:
- Missing extended audit logs for Azure Synapse Workspace
- No extended audit logs for Azure Synapse Workspace